lib/OpenQA/WebAPI/Plugin/ObsRsync/templates/ObsRsync_index.html.ep

```diff
@@ -33,6 +34,7 @@
                     % }
                     <td class="sync status"> <%= $folder_info->{state} %> </td>
                     <td class="job created"> <%= $folder_info->{created} %> </td>
+                    <td class="dirty status"> <%= $folder_info->{dirty_status} %> <%= link_post 'update' => url_for('plugin_obs_rsync_update_dirty_status', folder => $folder) %> </td>
```

Comment:
This seems wrong. Class names can not have whitespaces. In fact you're assigning the two classes dirty and status here. The class attributes of existing cells have the same problem. Since it seems these classes are not used anywhere I would simply remove them all.